### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by Uzlopak**
disable package-lock generation
remove got as devDependency and use instead fastify.inject
test again node 18
remove typescript as devDependency
lint in ci

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "plugin.js",
   "types": "index.d.ts",
   "scripts": {
+    "lint": "standard | standard",
```
Comment (Fdawgs) [inline package.json]: Sup with the double standard?

Comment (kibertoad) [reply package.json]: what's wrong with double standards? /troll

Comment (Uzlopak) [reply package.json]: I get it: less double standards, we need to be snazzy

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
CI is failing

**Comment by Uzlopak** [comment]
@kibertoad 

Fixed :)

**Review by kibertoad — APPROVED** [review]

**Pull request merged**